monkey/monkey_island/cc/repositories/file_agent_plugin_repository.py

```diff
@@ -0,0 +1,18 @@
+from common.agent_plugins import AgentPlugin, AgentPluginType
+
+from . import IAgentPluginRepository, IFileRepository, RetrievalError
+from .plugin_utils import load_plugin
+
+
+class FileAgentPluginRepository(IAgentPluginRepository):
+    def __init__(self, file_repository: IFileRepository):
+        self._file_repository = file_repository
+
+    def get_plugin(self, name: str, plugin_type: AgentPluginType) -> AgentPlugin:
```

Comment:
I've switched the order of these parameters on `develop`. This is essentially a filtering operation, which should go from least to most specific. You can either rebase on `develop` and fix this or cherry-pick 3f98f41f464 and fix it.

Cherry-pick is probably easier since this branch is based on another branch besides `develop`.